edit/namespace.vue

```diff
@@ -34,10 +34,10 @@ export default {
   mixins: [CreateEditView],
 
   async fetch() {
-    const projects = await this.$store.dispatch('management/findAll', { type: MANAGEMENT.PROJECT });
+    this.projects = await this.$store.dispatch('management/findAll', { type: MANAGEMENT.PROJECT });
     const projectName = this.value?.metadata?.labels?.[PROJECT] || this.$route.query[PROJECT_ID];
```

Comment:
Is it necessary to use this query twice? Would `this.projectName` in the `data()` object be enough?

Reply:
Fixed.